project-canvas/src/components/CreateIssue/Fields/PrioritySelect.tsx

```diff
@@ -27,24 +27,37 @@ export function PrioritySelect({
       ?.includes("Priority")
 
   return (
-    <Select
-      label="Priority"
-      placeholder="Choose priority"
-      nothingFound="Select an Issue Type first"
-      itemComponent={SelectItem}
-      disabled={isDisabled}
-      data={
-        !isLoading && priorities && priorities instanceof Array
-          ? priorities.map((priority) => ({
-              image: priority.iconUrl,
-              value: priority.id,
-              label: priority.name,
-            }))
-          : []
-      }
-      searchable
-      clearable
-      {...form.getInputProps("priority.id")}
-    />
+    <Tooltip
+      label="Priority cannot be selected for this issue type"
```

Comment:
when the issue type is not selected the message Priority cannot be selected for this issue type  is not clear . it should be choose an issue type first